components/detail/SwapSteps/SwapStepItemExpanded.tsx

```diff
@@ -30,25 +31,27 @@ function SwapStepItemExpanded(props: SwapStepItemExpandedProps) {
           const StepValueComponent = col.component;
           const { title, id } = col;
           return (
-            <>
-              <div
-                key={`column-${id}`}
-                className="flex flex-col md:grid md:grid-cols-7">
-                <div className="text-12 mb-5 md:mb-0 md:text-16 font-medium md:p-16 pl-0 md:col-span-3 text-primary-500 flex items-center">
-                  {`${title} : `}
+            ((id === 7 && !!step.failureReason) || id !== 7) && (
```

Comment:
NIT: instead of using id and multi-condition, you can do like this:

```
// Destructuring failureReason in top.
if (title === 'Failure Info' && !failureReason) return;
```

Reply:
fixed